logging/cluster-logging.adoc

```diff
@@ -2,32 +2,46 @@
+
+[NOTE]
+====
+The Kibana web console is deprecated and will be removed in a future logging release.
+====
+
+// Deployment
+{product-title} cluster administrators can deploy the {logging} by using Operators. For information, see xref:../logging/cluster-logging-deploying.adoc#cluster-logging-deploying[Installing the {logging-title}].
+
+The Operators are responsible for deploying, upgrading, and maintaining the {logging}. After the Operators are installed, you create a `ClusterLogging` custom resource (CR) to schedule {logging} pods and other resources necessary to support the {logging}.
+// info about log types TBA
+
 ifdef::openshift-enterprise,openshift-rosa,openshift-dedicated,openshift-webscale,openshift-origin[]
-As a cluster administrator, you can deploy the {logging} to aggregate all the logs from your {product-title} cluster, such as node system audit logs, application container logs, and infrastructure logs. The {logging} aggregates these logs from throughout your cluster and stores them in a default log store. You can xref:../logging/cluster-logging-visualizer.adoc#cluster-logging-visualizer[use the Kibana web console to visualize log data].
 
 [NOTE]
 ====
 Because the internal {product-title} Elasticsearch log store does not provide secure storage for audit logs, audit logs are not stored in the internal Elasticsearch instance by default. If you want to send the audit logs to the default internal Elasticsearch log store, for example to view the audit logs in Kibana, you must use the Log Forwarding API as described in xref:../logging/config/cluster-logging-log-store.adoc#cluster-logging-elasticsearch-audit_cluster-logging-log-store[Forward audit logs to the log store].
 ====
-endif::[]
 
-include::modules/logging-architecture-overview.adoc[leveloffset=+1]
+endif::[]
 
 ifdef::openshift-rosa,openshift-dedicated[]
 include::modules/cluster-logging-cloudwatch.adoc[leveloffset=+1]
 .Next steps
 * See xref:../logging/log_collection_forwarding/log-forwarding.adoc#cluster-logging-collector-log-forward-cloudwatch_log-forwarding[Forwarding logs to Amazon CloudWatch] for instructions.
 endif::[]
 
-include::modules/logging-common-terms.adoc[leveloffset=+1]
-include::modules/cluster-logging-about.adoc[leveloffset=+1]
+include::modules/logging-architecture-overview.adoc[leveloffset=+1]
+
+[id="cluster-logging-resources"]
+== Logging custom resources
 
-For information, see xref:../logging/cluster-logging-deploying.adoc#cluster-logging-deploying[Installing the {logging-title}].
+The `ClusterLogging` CR defines a complete {logging} environment that includes all the components of the logging stack to collect, store and visualize logs. The Red Hat OpenShift Logging Operator watches the {logging} CR and adjusts the logging deployment accordingly.
```

Comment:
So, the context here is that I'm just moving the content from https://github.com/openshift/openshift-docs/pull/64935/files#diff-4b32a605013f9ebca46ebb3d25aff9569d13c38bd6e44e6a2662a569f7b8be12L18 to this file, so that we can get rid of unnecessary additional files (modules) 😄 It was pre-existing content, probably referring to Elasticsearch.
I would say this is a larger issue that can be followed up as part of https://issues.redhat.com/browse/OBSDOCS-460 ?

Reply:
I defer to you here.  If it is "just a move" then I'm comfortable leaving as-is assuming we will correct

Reply:
Going to just undo this change for now and leave the module in until we have time to correct it. I don't want to add any confusion for reviewers.